openproblems/tasks/denoising/methods/knn_smoothing.py

```diff
@@ -0,0 +1,22 @@
+from ....tools.decorators import method
+from ....tools.utils import check_version

```

Comment:
Looks like you haven't used this. You'll need to add `check_version("knn_smooth")` to `adata.uns["method_code_version"]`

Reply:
Would that just entail adding one line in the knn_smoothing method that says:
`adata.uns["method_code_version"] = check_version("knn_smooth")`
?

Reply:
Precisely